source/release_notes/6.0.3-2.rst

```diff
@@ -0,0 +1,46 @@
+.. _PSMDB-6.0.3-3:
+
+================================================================================
+*Percona Server for MongoDB* 6.0.3-2 (2022-12-)
+================================================================================
+
+:Date: December , 2022
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/6.0/install/index.html>`_
+
+
+Percona Server for MongoDB 6.0.3-2 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for `MongoDB 6.0.3 Community edition <https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.3---nov-21--2022>`_. 
+
+|PSMDB| 6.0.3-2 fully supports MongoDB 6.0 protocols and drivers and does not require any code modifications.
+
+Release Highlights
+==================
+
+Improvements and bug fixes, provided by MongoDB and included in |PSMDB| are the following:
+
+* `SERVER-66289 <https://jira.mongodb.org/browse/SERVER-66289>`_ - Fixed the issue with how the server handles batches of writes when running $out with secondary read preference by updating write size estimation logic in ``DocumentSourceWriter``
+* `SERVER-68371 <https://jira.mongodb.org/browse/SERVER-68371>`_ - Allowed search queries to pass through query analysis when Client-Side Field Level Encryption is enabled for the MongoClient
+* `SERVER-68115 <https://jira.mongodb.org/browse/SERVER-68115>`_ - Prevented dropping empty path component from `elemMatch` path during index selection
+* `SERVER-68394 <https://jira.mongodb.org/browse/SERVER-68394>`_ - Prevented yielding strong locks upon startup recovery when ``_id`` index is missing
+
+ 
+Find the full list of new features and improvements in `MongoDB 6.0.3 Community Edition release notes <https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.3---nov-21--2022>`_.
+
+
+
+Improvements
+===========================
+
+* :jirabug:`PSMDB-1181`: Add backup cursor parameters to cursor's metadata
+  
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PSMDB-1082`: Improve error handling for |PSMDB| when the wrong encryption key is used
```

Comment:
I think PSMDB-1082 was part of the previious release

Reply:
Nope, it was in 5.0.13, not in 6.0.2. At least the fixed version field says that